ras_party/controllers/respondent_controller.py

```diff
@@ -125,17 +125,64 @@ def update_respondent_mark_for_deletion(email: str, session):
 @with_db_session
 def delete_respondents_marked_for_deletion(session):
```

Comment:
This controller method is invoked by a [batch HTTP endpoint](https://github.com/ONSdigital/ras-party/blob/main/ras_party/views/batch_request.py#L34-L41).

As nothing handled exceptions thrown in the above deletions, and the method doesn't throw exceptions, this makes troubleshooting errors impossible, and results in the user being told their account has been deleted. Technically the account is unusable as it will have deleted the auth record, but we need sight of party deletions issues here. Might need to alert on these log entries.